modules/FindWebP.cmake

```diff
@@ -0,0 +1,25 @@
+# - Find WebP
+# Find the WebP libraries
+#
+#  This module defines the following variables:
+#     WEBP_FOUND        - 1 if WEBP_INCLUDE_DIR & WEBP_LIBRARY are found, 0 otherwise
+#     WEBP_INCLUDE_DIR  - where to find webp/encode.h, etc.
+#     WEBP_LIBRARY      - the libwebp library
+
+find_path( WEBP_INCLUDE_DIR
+           NAMES webp/encode.h
+           PATH_SUFFIXES /usr/include /include
```

Comment:
Probably also a good idea to search for `/usr/local/include`? I am not sure though, just thinking.

Reply:
I'm new to cmake, so I used the same paths as in the other `Find*.cmake` files. If we used `/usr/local/` here, I feel like it'd also have to be changed for the other ones, which I think is a topic for another PR.

Reply:
I'm not a cmake veteran either :D
If `/local/` paths are not used in other `Find*.cmake` files, we should really not be bothered.